pages/human-donors/[uuid].js

```diff
@@ -94,11 +94,7 @@ export async function getServerSideProps({ params, req }) {
       : [];
     const phenotypeTermsList =
       phenotypicFeatures.length > 0
-        ? await request.getMultipleObjects([
-            ...new Set(
-              phenotypicFeatures.map((phenotype) => phenotype.feature)
-            ),
-          ])
+        ? phenotypicFeatures.map((phenotype) => phenotype.feature)
```

Comment:
Since this doesn’t do an igvfd request anymore, this code should move up to the `HumanDonor` UI component. I think the `getServerSideProps()` code should just request needed data and not just extract existing data, with few if any exceptions.

Reply:
Technically this will get deleted once the rodent donor changes go in which fix the phenotypic feature table, so I was using this as a quick placeholder. I think for now I'll just leave this as is until the other ticket merges.

Reply:
OK.

Reply:
Should we hold on this branch until the rodeo donor gets merged?